File: lista 15questoes/15 questoes em python/att07.py
class Escola:
    def __init__(self, nome):
        self.nome = nome
        self.professores = []

class Professor:
    def __init__(self, nome):
        self.nome = nome

escola1 = Escola("Escola A")
professor1 = Professor("Professor X")
escola1.professores.append(professor1)

File: lista 15questoes/15 questoes em java/att07.java
import java.util.ArrayList;
import java.util.List;

class Escola {
    private String nome;
    private List<Professor> professores;

    public Escola(String nome) {
        this.nome = nome;
        this.professores = new ArrayList<>();
    }
}

class Professor {
    private String nome;

    public Professor(String nome) {
        this.nome = nome;
    }
}

Escola escola1 = new Escola("Escola A");
Professor professor1 = new Professor("Professor X");
escola1.professores.add(professor1);
